### ISSUE TITLE
Chapter9. page 102, paragraph 2, confusing sentence

### ISSUE THREAD

**Issue by colwem** [crypto101/book#275]
> To put this into perspective, symmetric key algorithms work in order of magnitude 10 or so cycles per byte in either direction.

Does this mean "Symmetric key algorithms take an order of magnitude fewer cycles per byte in either direction" or does it mean "10 orders of magnitude fewer"  Or am I misunderstanding this entirely.

**Comment by lvh** [comment]
No, there's no "fewer"; this is saying that within an order or magnitude or so, symmetric key algorithms take about 10 cycles per byte. 1 cpb to 100 cpb covers most of the range between AES-NI + PCLMULQDQ-assisted AES-GCM or software Salsa20 (<10 cpb), to software 3DES (just over 100 or so cpb).

**Comment by colwem** [comment]
Oh ok then I would rephrase this to "symmetric key algorithms work **within an** order of magnitude **of** 10 or so cycles per byte in either direction.

**Comment by lvh** [comment]
OK, cool; adjusted that sentence. Thanks!
